village/src/index.js

```diff
@@ -1,7 +1,11 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
+import {BrowserRouter as Router} from 'react-router-dom';
```

Comment:
Antonio. This is a good import and alias convention. Could you explain what led you to this choice?

Reply:
From Gabes lecture on Routers its the main one that stuck to my head, that's why I used it